### ISSUE TITLE
Identify current cron queries and understand what each one does

### ISSUE THREAD

**Issue by jennlove-um** [tl-its-umich-edu/my-learning-analytics#1266]
From #489 
Decompose and understand each of the current queries that exist as part of the cron. These have evolved over time and some are likely inefficient and could be minimally improved.

Sam identified 8 queries that in 2019. I'd expect more might have been added since then.
Break up the parts a bit more.

**Comment by jennlove-um** [comment]
Sam identified in the following queries from `cron.py` that will need to be re-written to use data from the UDP. 

- [ ] A term query from `update_term`
- [ ] A course query from `verify_course_ids`
- [ ] A user query from `update_user`
- [ ] An assignment group query from `update_groups`
- [ ] An assignment query from `update_assignment`
- [ ] A submission query from `update_submission`
- [ ] A weight consideration query from `weight_consideration`
- [ ] A resource query from `update_canvas_resource`

Sam's notes: 
To help with this effort, I have started work in my [issue-489-set-up-cron-queries-refactor](https://github.com/tl-its-umich-edu/my-learning-analytics/compare/master...ssciolla:issue-489-set-up-cron-queries-refactor) branch. There I've included reformatted versions of these eight queries, along with the skeleton of a JSON file called `cron_queries.json`. This is mostly still the brainstorming stage (nothing is hooked up yet to the project), so if you like, don't like, or have ideas about this approach, let me know.
